lessons/intro/distribution/index.md

```diff
@@ -440,16 +391,14 @@ Tento soubor se dá vygenerovat z aktuálního prostředí zadáním
 `python -m pip freeze > requirements.txt` a balíčky v něm se dají nainstalovat
 pomocí `python -m pip install -r requirements.txt`.
 My ho používat nebudeme, vystačíme si s volnější specifikací závislostí
-v `setup.py`.
+v `pyproject.toml`.
 
 
 Nahrání na PyPI
 ---------------
 
-Balíček jde zaregistrovat a nahrát na PyPI. Původně k tomu sloužily příkazy
-`setup.py` `register` a `upload`, ale tyto příkazy používaly HTTP, což není
-bezpečné. Prototo je lepší použít program `twine` (instalovatelný přes pip),
-který používá HTTPS.
+Balíček jde zaregistrovat a nahrát na PyPI. Použijeme pro to program `twine`
+(instalovatelný přes pip).
 
 Budete si potřebovat zařídit
 [účet na PyPI](https://pypi.org/account/register/),
```

Comment:
A je potřeba vygenerovat si token. Do pypirc už se nedá dát jméno a heslo.

Reply:
Přidala jsem aktualizaci instrukci ohledně PyPI: rozhodla jsem se rozdělit instrukce na test PyPI (pracujeme s tím spolu na srazu) a ostré PyPI (vrať se k tomu, až budeš potřebovat). Představila jsem si chaos dvou účtů, dvou sad recovery kódů, dvou tokenů během fáze "rychle rychle registrujme" a přijde mi to lépe vstřebatelné v rozdělené podobě.